.github/workflows/python-tests.yml

```diff
@@ -32,6 +32,9 @@ jobs:
         pip install pylint~=2.17.7
         pip install -U ruff pytest pytest-doctestplus pytest-pylint pytest-mypy mock
         pip install -U types-pkg_resources
+        pip install --force-reinstall git+https://github.com/prusa3d/prusa-connect-sdk-printer.git
+        pip install --force-reinstall git+https://github.com/prusa3d/gcode-metadata.git
```

Comment:
nebylo by lepší tu verzi zahákovat tím hashem commitu? takhle se ti to bude instalovat pokaždý ne?